BrackeysBot.cs

```diff
@@ -62,7 +62,7 @@ public async Task Start()
             Data = new DataModule();
             Data.InitializeDataFiles();
 
-            Commands = new CommandHandler(Data, Configuration["prefix"]);
+            Commands = new CommandHandler(Data, Configuration["prefix"], Configuration["ignore_cc_prefix"]);
```

Comment:
What's this for? Why did this have to be done?

Reply:
"ignore_cc_prefix" is a safety check because you can accidentaly override important commands with Rexeg like "\w+"

Reply:
And what does "ignore_cc_prefix" setting contain?